configure.ac

```diff
@@ -51,4 +51,14 @@ PKG_CHECK_MODULES([libbase58],[libbase58])
 
 AC_CHECK_LIB([ws2_32], [strchr])
 
+#libgcrypt necessary to build example.c with "make example"
+AC_CHECK_LIB([gcrypt], [main],
+	AC_MSG_RESULT([\
+	libgcrypt available; compile example.c with "make example"])
+	[can_compile_example=yes],
+	AC_MSG_RESULT([\
+	libgcrypt unavailable; install it for compiling example.c with "make example"])
+	[compile_example=no])
+AM_CONDITIONAL([CAN_COMPILE_EXAMPLE], [test x$can_compile_example = xyes])
+
```

Comment:
According to [this](http://permalink.gmane.org/gmane.comp.sysutils.systemd.devel/10746), that macro is part of "libgcrypt-devel," presumably the developer additions to libgcrypt, which all users might not have.